roles/icinga_agent/defaults/main.yml

```diff
@@ -2,13 +2,20 @@
 icinga_agent_package: icinga2
 icinga_agent_registration: false
 icinga_agent_ca_host_icinga_port: 5665
-icinga_agent_constants_conf: |
+icinga_agent_constants_conf_rhel: |
```

Comment:
You used this beforfe:
```
icinga2_user:
  Debian: nagios
  RedHat: icinga
  ```
Can you change this var to:
```
icinga_agent_constants_conf:
  Debian:
  RedHat:
```

Then the if-condition in the template could be removed.

Reply:
Added the suggested fix. Thank you.